server/controllers/FavouriteController.js

```diff
@@ -0,0 +1,60 @@
+import models from '../database/models';
+import helpers from '../helpers';
+
+const { Author, Favourite } = models;
```

Comment:
Kindly update your favourite models to favouriteAuthor and table name to favouriteAuthors, because there is another model called favouriteBook

Reply:
I don't think that is necessary. The already existing 'Favourites' model was intended for this purpose; to be used for users and authors, so I think it's fine like that.